### PR TITLE
[Kotlin Cleanup] LanguageUtilsTest

### PR DESCRIPTION
**Pull request by oyeraghib**
## Purpose / Description
[Kotlin Cleanup] LanguageUtilsTest

## Checklist

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/test/java/com/ichi2/utils/LanguageUtilsTest.kt [hunk 1]
```diff
@@ -23,24 +23,22 @@ import com.ichi2.testutils.EmptyApplication
...
-@KotlinCleanup("Use Kotlin's HashSet instead of Java's")
-@KotlinCleanup("Use Kotlin's methods instead of Arrays.asList")
-@KotlinCleanup("`is` -> equalTo")
 class LanguageUtilsTest {
     @Test
     fun testNoLanguageIsRemoved() {
         val languages = HashSet<String?>()
         Collections.addAll(languages, *LanguageUtil.APP_LANGUAGES)
-        val previousLanguages: MutableList<String> = ArrayList(Arrays.asList(*PREVIOUS_LANGUAGES))
+        val previousLanguages: MutableList<String> = ArrayList(listOf(*PREVIOUS_LANGUAGES))
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/utils/LanguageUtilsTest.kt:41]:
```kotlin
        val languages = LanguageUtil.APP_LANGUAGES.toHashSet()
        val previousLanguages = PREVIOUS_LANGUAGES.toMutableList()
```

File: AnkiDroid/src/test/java/com/ichi2/utils/LanguageUtilsTest.kt [hunk 2]
```diff
@@ -49,7 +47,7 @@ class LanguageUtilsTest {
 
     @Test
     fun testCurrentLanguagesHaveNotChanged() {
-        val actual = Arrays.asList(*LanguageUtil.APP_LANGUAGES)
+        val actual = listOf(*LanguageUtil.APP_LANGUAGES)
```
Comment (david-allison) [inline AnkiDroid/src/test/java/com/ichi2/utils/LanguageUtilsTest.kt:50]: LanguageUtil.APP_LANGUAGES.toList()

**Review by david-allison — CHANGES_REQUESTED** [review]

**Comment by david-allison** [comment]
still waiting for feedback

**Review by mikehardy — APPROVED** [review]
all feedback addressed, thanks!

**Pull request merged**